public/app/features/alerting/unified/search/rulesSearchParser.ts

```diff
@@ -107,6 +115,9 @@ export function applySearchFilterToQuery(query: string, filter: RulesFilter): st
   if (filter.plugins) {
     filterStateArray.push({ type: terms.PluginsToken, value: filter.plugins });
   }
+  if (filter.ruleSource) {
+    filterStateArray.push({ type: terms.FreeFormExpression, value: `source:${filter.ruleSource}` });
```

Comment:
I think this should be something like
```suggestion
    filterStateArray.push({ type: terms.RuleSourceToken, value: filter.ruleSource });
```